e2e/helpers/auth-helpers.ts

```diff
@@ -0,0 +1,34 @@
+import { type Page } from "@playwright/test";
+
+export class AuthHelpers {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async loginJujuLocal() {
```

Comment:
The auth methods aren't tied to the environment, so this can just be:

```suggestion
  async loginLocal() {
```